guides/integrate-firebase.md

```diff
@@ -0,0 +1,272 @@
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SetupKotlin from '/docs/workflows/_sdk-setup/_setup-kotlin.mdx';
+import SetupSwift from '/docs/workflows/_sdk-setup/_setup-swift.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+
+# Integrate Beyond Identity Passwordless Authentication into Firebase
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for an application that uses [Firebase](https://firebase.google.com/docs/auth)
+
+This guide will cover:
+
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+- How to configure the OpenID Connect plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## Firebase
+
+- [Authenticate Using OpenID Connect on Android](https://firebase.google.com/docs/auth/android/openid-connect)
+- [Authenticate Using OpenID Connect on Apple platforms](https://firebase.google.com/docs/auth/ios/openid-connect)
+
+## Install the Embedded SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android) and the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+## Initialize the Embedded SDK
+
+Once you've installed the SDK, initialize it so that you can call the Embedded functions.
+
+<Tabs groupId="sdks" queryString>
+<TabItem value="kotlin" label="Kotlin">
+
+<SetupKotlin />
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+<SetupSwift />
+
+</TabItem>
+</Tabs>
+
+## Set up Beyond Identity as an Identity Provider
+
+To set up Beyond Identity as an Identity Provider, you need to create a [Realm](/docs/v1/workflows/realms) to hold identities and configuration. Inside that realm, you'll need to create an [Application](/docs/v1/workflows/applications) that contains the authentication flow configuration. These can be configured in you admin console that was created for you when you signed up for a developer account.
+
+### Create a Realm
+
+From the admin console, click the realm drop-down, and click **Create new realm**.
+
+<Arcade clip={Clip.CreateRealm} />
+
+### Create an Application
+
+From the admin console, click **Apps**, then click **Add app**.
+
+There is a lot to configure when creating an application. When creating your application make sure:
+
+- Redirect URIs contains your application's App Scheme or Universal URL
+- Configuration Type (in the Authenticator Config tab) is set to `Embedded SDK`
+- Invoke URL (in the Authenticator Config tab) contains your application's App Scheme or Universal URL
+- Invocation Type (in the Authenticator Config tab) is set to `Automatic`.
```

Comment:
Did you test different configurations on the `Token Configuration Subject`. Like should it be set to "id"?

Reply:
Do you mean this?

<img width="835" alt="Screenshot 2023-05-15 at 4 51 43 PM" src="https://github.com/gobeyondidentity/developer-docs/assets/4763133/ec714bb4-0b49-49f5-9b13-f623483e649a">

I actually only tested with email. Let me test with id.

Edit: It worked with email or id. I didn't notice any difference in the User tab in Firebase Console, so I think it is using the User Info endpoint to populate the data.

<img width="936" alt="Screenshot 2023-05-15 at 4 59 01 PM" src="https://github.com/gobeyondidentity/developer-docs/assets/4763133/730ba035-a03b-4ba2-b41a-ce76cd712604">

Reply:
ok cool! Yea wordpress requires using "id".